Mergin/validation.py

```diff
@@ -427,6 +467,14 @@ def warning_display_string(warning_id):
     elif warning_id == Warning.MERGIN_SNAPPING_NOT_ENABLED:
         return "Snapping is currently enabled in this QGIS project, but not enabled in Mergin Maps Input"
     elif warning_id == Warning.MISSING_DATUM_SHIFT_GRID:
-        return "Required datum shift grid is missing, reprojection may not work correctly. <a href='#fix_datum_shift_grids'>Fix the issue.</a>"
+        return "Required datum shift grid is missing, reprojection may not work correctly. <a href='fix_datum_shift_grids'>Fix the issue.</a>"
     elif warning_id == Warning.SVG_NOT_EMBEDDED:
         return "SVGs used for layer styling are not embedded in the project file, as a result those symbols won't be displayed in Mergin Maps Input"
+    elif warning_id == Warning.EDITOR_PROJECT_FILE_CHANGE:
+        return f"You don't have permission to edit QGS project file. Ask the workspace admin to upgrade your permission or <a href='{url}'>reset QGS project file</a> to be able to sync data changes. This might involve deleting layers you created locally."
```

Comment:
How about something like this? We need to adjust these error messages to what is actually happening (not only this one but other too - see https://github.com/MerginMaps/qgis-plugin/issues/581#issuecomment-2168512692 and the following conversation)
```suggestion
        return f"You don't have permission to edit the QGIS project file. Your changes to this file will not be sent to the server. Ask the workspace admin to upgrade your permission if you want your changes sent to the server. You can also <a href='{url}'>restore this QGIS project file</a> to the server version."
```